### DISCUSSION TITLE
Add a way to search for maps that needs to be updated

### DISCUSSION THREAD

**Discussion by Arnacino** [ppy/osu#21633] [Ideas]
I'm actually not 100% sure that there's a way or not, but it would be cool to have something like status=u (?) or something to show up only maps that need to be updated. While I'm here is there also a way to update all updatable maps with just one button?

**Comment by Flamiii** [comment]
A button to update all beatmaps will likely come in the future. https://github.com/ppy/osu/discussions/19572#discussioncomment-3319268

**Reply by Arnacino** [reply]
didn't look for that, just thought it while writing the comment, thanks!
